objectbox/lib/src/modelinfo/modelproperty.dart

```diff
@@ -23,7 +23,7 @@ class ModelProperty {
 
   set name(String? value) {
     if (value == null || value.isEmpty) {
-      throw Exception('name must not be null or an empty string');
+      throw ArgumentError('name must not be null or an empty string');
```

Comment:
Maybe use `ArgumentError.value(...)` in this file, too? But I guess it just affects formatting of the output and does not provide source code linking or similar.

Reply:
Nah, AFAIK it just concatenates the string. This is fairly internal error anyway so as long as the model isn't messed up, should never really happen